recipes/cockroach/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "cockroach" %}
+{% set version = "19.2.5" %}
+
+package:
+  name: go-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://binaries.cockroachdb.com/{{ name }}-v{{ version }}.src.tgz
+  sha256: 7908fcf2dfe9fc525b3a5f60983372ab61dc859f28320bd23b2d9afb77cf31bb
+  patches:
+    - 0001-use-conda-forge-dependencies.patch
+
+build:
+  number: 0
+  #pin_depends: strict
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler("go-cgo") }} >=1.12.17,<1.13
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - cmake >=3.8,<4
+    - protobuf ==3.6.0
+    - yarn
+  host:
+    - jemalloc
+    - libprotobuf
+    - libedit
+    - rocksdb
+  run:
+    - {{ pin_compatible('jemalloc', max_pin="x") }}
+    - {{ pin_compatible('libedit', max_pin="x.x") }}
+
+test:
+  commands:
+    - cockroach version
+    - cockroach start-single-node --insecure --background
+    - cockroach workload init startrek
+    - cockroach sql --insecure -d startrek -e 'select quote from quotes limit 1;'
+    - cockroach quit --insecure
+
+about:
+  home: http://cockroachlabs.com
+  license: BSL-1.1
+  license_file: src/github.com/cockroachdb/cockroach/licenses/BSL.txt
+  summary: 'A disaster-surviving, cloud-native SQL database for building global, scalable cloud services.'
+  description: |
+    CockroachDB is a distributed SQL database built on a transactional and
+    strongly-consistent key-value store. It scales horizontally; survives disk,
+    machine, rack, and even datacenter failures with minimal latency disruption and
+    no manual intervention; supports strongly-consistent ACID transactions; and
+    provides a familiar SQL API for structuring, manipulating, and querying data.
+  doc_url: https://www.cockroachlabs.com/docs
+  dev_url: https://github.com/cockroachdb/cockroach
+
+extra:
+  recipe-maintainers:
+    - sodre
```

Comment:
```suggestion
    - benhuff
    - oblute
    - rluria14
    - sodre
```

Reply:
@benhuff, @oblute, @rluria14 do you agree to be maintainers of `cockroachdb`?

Reply:
yes, i agree to be a maintainer

Reply:
i agree also 👍

Reply:
sounds good 👍